### ISSUE TITLE
UniversalViewer does not display content consistently in some cases

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/orangelight#1512]
https://catalog-staging.princeton.edu/catalog/8162646 does not render the UV for @escowles:
![screen shot 2018-12-13 at 12 55 18 pm](https://user-images.githubusercontent.com/1443986/49957833-09c30680-fed7-11e8-8c78-2a06e12441e2.png)

For me, this was only rendered inconsistently for a random number of attempts to view the entry.

**Comment by christinach** [comment]
The problem is that we are loading the `uv-modal` 3 times in the `show page`. The uv-modal has the same `id` and this is causing the viewer not be displayed consistently. I'm assuming we're loading the modal in 3 different areas for testing purposes and to get feedback on the show page. If we are planning to trigger the modal from more than one areas in the show page then we must update the code and give a `unique modal id`.

**Comment by christinach** [comment]
I created a branch https://github.com/pulibrary/orangelight/tree/1512-uv-not-consistent with 2 modals (a thumbnail modal and a digital modal) and unique ids, and locally I generated unique random iframe ids but I still find the inconsistency in displaying the full content of the viewer. ->

What happens is that if you trigger the modal (from the thumbnail or from the Digital Content link) before the viewer (at the bottom of the page) fully loads then you are able to see the full content in one of the modals (the one you triggered first). 
If you trigger the modal after the viewer (at the bottom of the page) is fully loaded then you are not able to see the full content of the viewer in any of the modals. 
This is why I followed the approach to create unique ids for the modals and the iframes but so far that has not fixed the issue.
